src/assets/js/profile-api.js

```diff
@@ -0,0 +1,21 @@
+import Fetcher from '@/assets/js/fetcher';
```

Comment:
I guess that's not needed anymore?

Reply:
True, my intention is to move away from that, but I'd rather not remove current code at the moment. I'd rather just add it there, and once we merge, then do another PR for cleaning up the api. Personal preference?

Reply:
I Think this not referred anywhere, therefore just remove that one file?